src/main/java/frc/robot/OI.java

```diff
@@ -44,39 +85,96 @@ public static Supplier<Double> getRotationSupplier(){
         }
 
         public static JoystickButton getOrientationButton(){
-            return new JoystickButton(kJoystick, kOrientationButtonID);
+            kOrientationButton.setButtonName("Orientation Button");
+            return new JoystickButton(kJoystick, kOrientationButton.getButtonID());
         }
 
         public static JoystickButton getZeroButton(){
-            return new JoystickButton(kJoystick, kZeroButtonID);
+            kZeroButton.setButtonName("Zero Button");
+            return new JoystickButton(kJoystick, kZeroButton.getButtonID());
         }
 
         public static JoystickButton getAlignForwardButton(){
-            return new JoystickButton(kJoystick, kAlignForwardButtonID);
+            kAlignForwardButton.setButtonName("Align Forward");
+            return new JoystickButton(kJoystick, kAlignForwardButton.getButtonID());
         }
 
         public static JoystickButton getAlignBackButton(){
-            return new JoystickButton(kJoystick, kAlignBackwardButtonID);
+            kAlignBackwardButton.setButtonName("Align Backward");
+            return new JoystickButton(kJoystick, kAlignBackwardButton.getButtonID());
         }
 
         public static JoystickButton getIntakeButton(){
-            return new JoystickButton(kJoystick, kIntakeButtonID);
+            kIntakeButton.setButtonName("Intake Button");
+            return new JoystickButton(kJoystick, kIntakeButton.getButtonID());
         }
         
         public static JoystickButton getOuttakeButton(){
-            return new JoystickButton(kJoystick, kOuttakeButtonID);
+            kOuttakeButton.setButtonName("Outtake Button");
+            return new JoystickButton(kJoystick, kOuttakeButton.getButtonID());
         }
     }
 
     public static final class Operator{
+        public enum OperatorButtons {
```

Comment:
Not sure there is a good reason to have 2 separate enum definitions. A single OIButtons enum seems like it would work fine

Reply:
If we decide to use differing controllers, it's good to have the separate enums